docs/decisions/infra/2023-05-25-provision-database-users-with-serverless-function.md

```diff
@@ -1,25 +1,25 @@
 # Provision database users with serverless function
 
-* Status: proposed
```

Comment:
Several ADRs still list their status as "proposed" rather than accepted. Is this accurate, out of curiosity?